src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -256,7 +256,9 @@
             aggregate-fields)])))})
 
 (def ^:private PERSISTENCE-KEY "library-page")
-(def ^:private VERSION 3)
+(def ^:private VERSION 2)
+(def ^:private TABLE_VERSION 1)
+
```

Comment:
why did `VERSION` go down? And where is `TABLE_VERSION` used?

Also, let's coordinate version numbers between this and #680.